packages/backend/server/tests/nestjs/throttler.spec.ts

```diff
@@ -156,7 +156,7 @@ test('should use default throttler for unauthenticated user when not specified',
 
   t.is(headers.limit, '120');
   t.is(headers.remaining, '119');
-  t.is(headers.reset, '60');
+  t.regex(headers.reset, /59|60/);
```

Comment:
extract all header assert as a function?
and we can compare number in range to make test more stable

Reply:
This should always be an integer string. so this should be enough imo